demo.py

```diff
@@ -0,0 +1,22 @@
+import streamlit as st
+from ner.model import extract_entities
+
+txt = st.text_area('Text to analyze')
+
+result = extract_entities(txt)
+
+st.write('Entities recognized:')
+result_dict = {}
+if result:
```

Comment:
my personal preference is to be explicit with your intentions and check for `result == []`, but this is fine